pipelines/tekton/aiedge-e2e/tasks/kserve-download-model.yaml

```diff
@@ -17,13 +17,16 @@ spec:
     description: The S3 URL used to download the model
   steps:
   - name: download-model-s3
-    image: quay.io/opendatahub/kserve-storage-initializer:v0.11
+    image: quay.io/opendatahub/kserve-storage-initializer:v0.11.1.3 
     script: |
       mkdir -p $(workspaces.workspace.path)/model_dir-$(params.model-name)/
 
       export S3_URL="s3://$(params.s3-bucket-name)/$(params.model-name)"
       echo -n $S3_URL | tee $(results.s3-url.path) ;
 
+      export CA_BUNDLE_CONFIGMAP_NAME=ssl-ca-directory
```

Comment:
Does this actually query for a ConfigMap named `ssl-ca-directory`?  If so, is there a kserve documentation about what is expected for the ConfigMap?

Reply:
To answer my own question: From [kserver v11.1.3/storage.py](https://github.com/opendatahub-io/kserve/blob/v0.11.1.3/python/kserve/kserve/storage/storage.py#L183), it looks like non-null `CA_BUNDLE_CONFIGMAP_NAME` is the logic check for getting the custom cert path in `AWS_CA_BUNDLE` and in this storage-initializer context there is no k8s client reading an actual ConfigMap